time.go

```diff
@@ -25,5 +29,9 @@ func (t Time) MarshalJSON() ([]byte, error) {
 }
 
 func (t Time) String() string {
+
```

Comment:
I don't think String() should return "". Rather call time.Time.String(), e.g. https://play.golang.org/p/TC86xxBDG9-.

Reply:
I am not sure I get it. I believe returning time.Time.String() might create problem when request is unmarshalled at end points since int64 is expected but a string will be retrieved.

Other option would be rename the function (ie, Int64 and return "", or 0 depending on the return type).

Reply:
String() returns a string representation of the time, which shouldn't really ever be "". That's not very useful for someone who's trying to log the time, for example. But for the query string case, we do want a zero time to convert to an empty string, so another function is needed.